terms/clinical/apoeGenotype.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "sage.annotations-clinical.apoeGenotype-0.0.1",
+    "description": "Apolipoprotein E genotype",
+    "source": "https://www.genecards.org/cgi-bin/carddisp.pl?gene=APOE",
+    "anyOf": [
+        {
+            "const": "22",
+            "description": "E2E2"
+        },
+        {
+            "const": "23",
+            "description": "E2E3"
+        },
+        {
+            "const": "24",
+            "description": "E2E4"
+        },
+        {
+            "const": "33",
+            "description": "E3E3"
+        },
+        {
+            "const": "34",
+            "description": "E3E4"
+        },
+        {
+            "const": "44",
+            "description": "E4E4"
+        }
+    ]
+}
```

Comment:
These will need a source (#682). Should we just have the gene card as the source for each of them?

Reply:
The gene card was originally the source for each of them, but I had thought I recalled that there had been discussion for other terms of making the individual item source into the key source if the individual item source was the same for all items.  If this is not the case, I can revert it to have the gene card on each possible value.

Reply:
Ah, I vaguely remember that conversation now, as well. I completely agree that it makes sense as a key source. However, based on discussion about #682, we will probably require sources (unless you want to weigh in with your thoughts) for all enumerated values (this makes the dictionary apps and AD templates more useful). Due to that, the source will need to be moved back to being on the individual values.